spec/system/signup_spec.rb

```diff
@@ -103,7 +104,7 @@
 
     it "redirects to a new-topic after signin and keeps the query" do
       category = Fabricate(:category)
-      SiteSetting.default_trust_level = 4
```

Comment:
This was preventing the user from being able to create a topic and thus unable to open the composer.